samples/plugins/actian_odbc/dialect.tdd

```diff
@@ -482,7 +482,8 @@
 		<argument type='str' />
     </function>
     <date-function name='DATEDIFF' return-type='int'>
-		<formula>TIMESTAMPDIFF(SECOND,CAST(%2 AS TIMESTAMP(4)),%3)/86400/7</formula>
+		<formula>CAST(ROUND(TIMESTAMPDIFF(DAY,%2,%3)/7.00,0) AS iNTegEr)</formula> <!--Sunil : Added Formula using DAY-->
```

Comment:
1. remove the comment and make it an entry in the commit description
2. use consistent case on the type unless there are syntax reasons
3. I'm not keen on literals with what appear to be spurious digits. I'm assuming you need the math to be on a decimal with 2 points of fractional precision but this is not documented anywhere, nor is the reasoning for it. I'm sure there are good reasons so they should be documented. If decimal with 2 significant digits for the fraction is needed recommend using `decimal(7, 3, 2)`

Reply:
**use consistent case on the type unless there are syntax reasons** - Used mix type to distinguish between generated SQL for failed cases but was for testing purpose only so needs to be removed from the version to be sent in PR.